drop-server.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/sh
+
+pythonpath=$(command -v python2 || command -v python)
+startstopdeamon="/sbin/start-stop-daemon"
```

Comment:
Ok at least `start-stop-daemon` is not required. Which does not make the `lsof` hack better but anyway.